media-sound/jack/metadata.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+<maintainer type="person">
```

Comment:
in general, every tag level needs an indent. Leave this, I will fix it on merging